intellij-ide.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: page
+title: Developing apps in the IntelliJ IDE
+sidebar: home_sidebar
+permalink: /intellij-ide/
+---
+
+Our custom [IntelliJ IDEA](https://www.jetbrains.com/idea/) plugins provides a
+fully integrated development experience in the IntelliJ IDE.
+
+* TOC Placeholder
+{:toc}
+
+## Installation and setup
+
+Please follow the [Flutter setup](/setup/) instructions to install IntelliJ and
+the Dart and Flutter plugins.
+
+### Updating the plugins<a name="updating"/>
+
+Updates to the plugins will be shipped on a regular basis. You should get
+prompted in IntelliJ when an update is available.
+
+To check for updates manually:
+
+1. Open preferences (**IntelliJ IDEA>Check for Updates...** on macOS, **Help>Check for Updates...** on Linux)
+1. If `dart` or `flutter-intellij` are listed, update them
+
+## Creating projects
+
+### Creating a new project from templates
+
+To create a new Flutter IntelliJ project with a fresh set of Flutter template
+files:
+
+1. In IntelliJ, click **Create New Project** from the 'Welcome' window or
+**File>New>Project...** from the main IDE window
+1. Select **Flutter** in the menu, and click **Next**
+1. Enter your desired **Project name** and **Project location**
+1. Click **Finish**
+
+### Creating a new projects from existing source code
+
+To create a new Flutter IntelliJ project containing existing Flutter source code
+files:
+
+1. In IntelliJ, click **Create New Project** from the 'Welcome' window or
+**File>New>Project...** from the main IDE window
+  - **Note**: Do *not* use the **New>Project from existing sources...** option for Flutter projects
+1. Select **Flutter** in the menu, and click **Next**
+1. Under **Project location** enter, or browse to, the directory holding your existing Flutter source code files
+1. Click **Finish**
+
+## Editing code, and viewing code problems
```

Comment:
Screenshots: yes, need that here and in a few other places. Not sure I will get to them for this first changeset though